p2p/src/authenticated/mod.rs

```diff
@@ -198,10 +198,13 @@
 //!
 //! // Configure network
 //! //
+//! // In production, use a unique application namespace to prevent cryptographic replay attacks.
 //! // In production, use a more conservative configuration like `Config::recommended`.
```

Comment:
nit: Now that we configure a few more things prior to config, I suggest we break apart this block and use this comment above `authenticated::Config::aggressive`